.cargo/config.toml

```diff
@@ -1,3 +1,6 @@
 [build]
 target="wasm32-wasi"
-rustflags="-C target-feature=+bulk-memory"
\ No newline at end of file
+rustflags="-C target-feature=+bulk-memory"
+
+[target.wasm32-wasi]
+runner="timeout 60s wasmedge --dir=.:. "
```

Comment:
Can it return a error when timeout?
If some bug let test timeout, CI will get this error?

Reply:
I not sure. In my local, it won't timeout. If timeout, it just be killed without throwing any error.

Reply:
I think need delete `timeout 60s`. because it will hide some bug.

Reply:
FYI, use the workflow timeout instead: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepstimeout-minutes

Reply:
You can set the expected timeout of the current step. If it really takes so long, then letting it fail is fine.

Reply:
If some test blocked for some reason, it's need 6h to wait github action to kill it. `timeout 60s` is just make it fail faster.

Reply:
I will change it to github action's timeout.